packages/common/src/error.rs

```diff
@@ -301,6 +305,31 @@ pub enum ContractError {
 
     #[error("Verification Failed")]
     VerificationFailed {},
+
+    #[error("Channel doesn't exist: {id}")]
+    NoSuchChannel { id: String },
+
+    #[error("You can only send cw20 tokens that have been explicitly allowed by governance")]
+    NotOnAllowList,
+
+    #[error("Didn't send any funds")]
+    NoFunds {},
```

Comment:
Nit: We already have a few errors that seem to fit this (InsufficientFunds and InvalidFunds). It would be better to use the existing ones.